### ISSUE TITLE
Suggest to Enable Neo-cli Single Command Executable

### ISSUE THREAD

**Issue by cloud8little** [neo-project/neo-node#588]
**Summary or problem description**
Currently we can only input command after starting neo-cli, such as create/open wallet, deploy contract, invoke contract, and the same time, neo-cli will sync blocks in one process.

![37a2fb45bb72dadba950550e78191c9](https://user-images.githubusercontent.com/34291844/82290998-b5308e80-99da-11ea-90b6-6667cbef9b7d.png)

The idea is to split neo-cli into two separate part: (temporary) Node and Interactive Console.
Node: sync block when new block comes, deal with p2p connection. this is a process and similar to the "server" side.
Interactive Console: accept user command similar to RPC request, such as **./neo-cli openwallet "1.json"** and forward the request to the Node and Node will return the response to the Console, then print the output on the console. 

Advantage: no need to screen/nohup start neo-cli at the beginning. and it is easier to input a command than a whole RPC request, which will need extra request header and body.

**Neo Version**
- Neo 3

**Where in the software does this update applies to?**
- CLI

**Comment by Tommo-L** [comment]
I think it's a good feature, if so, we can split into three things:
An executeable neo-node, which can accept some startup parameters.
An interactive and simple neo-cli, which communicate with neo-node by rpc or ipc, can also connect a remote node.
An interactive and simple neo-gui, which communicate with neo-node by rpc or ipc, can also connect a remote node.

And when we release neo-node, we can also package neo-cli and neo-gui together.

**Comment by doubiliu** [comment]
Agree, this can simplify the function of the node.But connecting remote nodes may need to consider some security issues。
